### DISCUSSION TITLE
Czech translation discussions

### DISCUSSION THREAD

**Discussion by joyinko** [ihhub/fheroes2#6900] [General]
@ihhub @zenseii 
I opened this discussion based on request in one of my PRs, we need some room for sharing the ideas and perhaps splitting the job as GoergeK1ng has offered helping hand. 
I searched for previous discussions, did not find one.
I'd like to held this discussion "room" in Czech language.

@GeorgeK1ng , zde můžeme diskutovat o příštích aktualizacích pro Český překlad hry.

**Comment by GeorgeK1ng** [comment]
Super, jak dokončíš aktuální [PR,](https://github.com/ihhub/fheroes2/pull/6893) tak na to vlítneme. Někdy před rokem jsem requestnul support pro češtinu, později se přidal font, ale neměl jsem na to čas. Zkoušel jsem přemluvi Gremlina (autor HoMM češtin), ale nechtělo se mu... 

EDIT: V mezičase jsem uploadnul aktualizovaný skript na správnou detekci HoMM II GOLD CZ (a ostatních Legacy instalací).

**Reply by joyinko** [reply]
@GeorgeK1ng 
Zenseii mi ten aktuální PR již schválil, takže už jen jeden krok a může být sjednocen.
Na co se plánuješ zaměřit? 
Já měl teď v plánu průvodní texty ke kampani datadisku. Tak to můžeš nechat na mně.
Až vytvořiš svůj PR prosím napiš mi to sem anebo v PR jako zmínku, ať vím co jsi upravoval.

**Reply by GeorgeK1ng** [reply]
Supr, chtěl bych vědět odkud mám začít. Můžeme si říct nějaký point (řádek, id) od kterého pojedu, ať to máme rychleji.
Překládáš ty texty manuálně nebo je umíš nějak brát z původní hry a upravit a napasovat na nový mechanismus?

**Reply by joyinko** [reply]
> Supr, chtěl bych vědět odkud mám začít. Můžeme si říct nějaký point (řádek, id) od kterého pojedu, ať to máme rychleji. Překládáš ty texty manuálně nebo je umíš nějak brát z původní hry a upravit a napasovat na nový mechanismus?

Velké okno je mezi řádky 4806-5830, to by mělo být na 4 až 5 PRs.

**Reply by joyinko** [reply]
@GeorgeK1ng zdravím
PR #6893 byl již sloučen, tak řekni zda budeš teď vytvářet nový?
Deadline pro inputy k 1.0.3 rel je sedmého dubna. 
Díky

**Reply by GeorgeK1ng** [reply]
Zatím jsem nezačal tvořit. :)

Vlítnu na to nejdříve s 1.0.3

**Comment by GeorgeK1ng** [comment]
@joyinko Zkusil jsem sestavit aktuální branch a vidím ZRUSIT místo ZRUŠIT. Jak to vypadá u tebe?

![image](https://user-images.githubusercontent.com/98261225/228018847-d1d2e1bc-fdac-49eb-aa27-e8f1f9d9b58b.png)

Stejně to vidím tady, že by problém s fontem?

![image](https://user-images.githubusercontent.com/98261225/228019749-4d92828e-f46c-453c-ac29-a2847f3c566e.png)

**Reply by joyinko** [reply]
![Snímek obrazovky_2023-03-27_19-59-48](https://user-images.githubusercontent.com/123939177/228027358-e529968b-0eb0-4246-ae39-dd2b2616109f.png)
@GeorgeK1ng 
Na tlačítkách to není správně, ostatní texty u mně OK, používám ale cs.mo z mého PR ne ofiko z 1.0.2 rel.

**Reply by GeorgeK1ng** [reply]
@joyinko Jo stáhnu si ten z tvého repository

**Reply by joyinko** [reply]
@GeorgeK1ng 
Jestě jsem to zkontroloval a ve Win 10 je to stejně i u mně. 
V Linuxu jsou fonty ok ale ty tlačítka je potřeba pořešit asi napříč všemi OS.

**Comment by GeorgeK1ng** [comment]
Vypadá to na problém, založil jsem ticket https://github.com/ihhub/fheroes2/issues/6906
